map.css

```diff
@@ -90,6 +90,24 @@ html, body, #map {
 		margin-right: 10px;
 	}
 }
+
+.panel .panel-content {
+	overflow-y: auto;
+    position: absolute;
+    width: calc(100% - 5px);
```

Comment:
Why won't you remove that 5px padding from parent and simply move it here? This way you would not need that `.panel .panel-content > ...` classes at all.